openmdao/drivers/scipy_optimizer.py

```diff
@@ -544,22 +561,23 @@ def accept_test(f_new, x_new, f_old, x_old):
 
         self.result = result
 
-        if hasattr(result, 'success'):
-            self.fail = False if result.success else True
-            if self.fail:
-                print('Optimization FAILED.')
+        if comm.rank == 0:
```

Comment:
You don't want to change how we set `self.fail` on any rank because that may be being used to tell some outer-level problem or workflow to continue.  The story is just to turn off the output printing, so the simplest thing is just to put an extra `if` check around the prints.